### DISCUSSION TITLE
SWInspector

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#176] [Show and tell]
I finally got around to sharing my level editing tool. I wouldn't get your hopes up too much, it's not easy to use and has obvious limitations, but it works and it's something. I've already made a few things with it. At the very least it's useful to understand what is in a level and what an NPC is trying to do.

https://github.com/Moburma/SWInspector

**Comment by geist22** [comment]
Checking it out right now. Yeah it looks somewhat complicated, I doubt I'll be able to make any edits myself. But for obtaining information about how the missions work, this will be useful.

Is it normal for the maps to be inverted compared to how they look ingame?

**Reply by Moburma** [reply]
> Is it normal for the maps to be inverted compared to how they look ingame?

Yes, confusingly that's how it works, this includes Netscan info as well, it's all reversed coordinate wise.
